### PR TITLE
Fix separator for last item in group

### PR DESCRIPTION
**Pull request by judastheo**
## Checklist before requesting a review
- [x] Working on iOS
- [x] Working on Android
- [x] Integration tests added
- [x] Visual regressions screenshots are up to date
- [ ] Design validated

### CODE REVIEW

File: packages/sandbox/e2e/android/screenshots/ListItemGroup.png
Comment (judastheo) [inline packages/sandbox/e2e/android/screenshots/ListItemGroup.png:1]: @ianrobertlambert no longer the separator on the last item!

File: packages/sandbox/e2e/ios/screenshots/ListItemGroup.png
Comment (judastheo) [inline packages/sandbox/e2e/ios/screenshots/ListItemGroup.png:1]: @ianrobertlambert no longer the separator on the last item!

Comment (ianrobertlambert) [reply packages/sandbox/e2e/ios/screenshots/ListItemGroup.png:1]: Great, thanks @judastheo ! 🙌 👌

**Review by kimak — APPROVED** [review]

**Comment by ianrobertlambert** [comment]
Thanks @judastheo !! ✨ Looks good to me.

**Pull request merged**